pkg/crc/constants/constants.go

```diff
@@ -24,6 +24,7 @@ const (
 	ConfigFile           = "crc.json"
 	LogFile              = "crc.log"
 	GlobalStateFile      = "globalstate.json"
+	/* #nosec */
 	DefaultPullSecretURL = "https://cloud.redhat.com/openshift/install/crc/installer-provisioned"
```

Comment:
Does this work if it's added at the end of the line? This would be less disruptive when reading the code
`DefaultPullSecretURL = "https://cloud.redhat.com/openshift/install/crc/installer-provisioned" /* #nosec */`

Reply:
```diff
+
+	/* #nosec */
	DefaultPullSecretURL = "https://cloud.redhat.com/openshift/install/crc/installer-provisioned"
```
What about an additional whiteline to indicate a block?

Reply:
@gbraad that is just a convention as put in https://github.com/securego/gosec#annotating-code here.